homeassistant/components/knx/schema.py

```diff
@@ -242,26 +244,37 @@ class CoverSchema:
     DEFAULT_TRAVEL_TIME = 25
     DEFAULT_NAME = "KNX Cover"
 
-    SCHEMA = vol.Schema(
-        {
-            vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-            vol.Optional(CONF_MOVE_LONG_ADDRESS): ga_list_validator,
-            vol.Optional(CONF_MOVE_SHORT_ADDRESS): ga_list_validator,
-            vol.Optional(CONF_STOP_ADDRESS): ga_list_validator,
-            vol.Optional(CONF_POSITION_ADDRESS): ga_list_validator,
-            vol.Optional(CONF_POSITION_STATE_ADDRESS): ga_list_validator,
-            vol.Optional(CONF_ANGLE_ADDRESS): ga_list_validator,
-            vol.Optional(CONF_ANGLE_STATE_ADDRESS): ga_list_validator,
-            vol.Optional(
-                CONF_TRAVELLING_TIME_DOWN, default=DEFAULT_TRAVEL_TIME
-            ): cv.positive_float,
-            vol.Optional(
-                CONF_TRAVELLING_TIME_UP, default=DEFAULT_TRAVEL_TIME
-            ): cv.positive_float,
-            vol.Optional(CONF_INVERT_POSITION, default=False): cv.boolean,
-            vol.Optional(CONF_INVERT_ANGLE, default=False): cv.boolean,
-            vol.Optional(CONF_DEVICE_CLASS): cv.string,
-        }
+    SCHEMA = vol.All(
+        vol.Schema(
+            {
+                vol.Required(
+                    vol.Any(CONF_MOVE_LONG_ADDRESS, CONF_POSITION_ADDRESS),
+                    msg=f"At least one of '{CONF_MOVE_LONG_ADDRESS}' or '{CONF_POSITION_ADDRESS}' is required.",
+                ): object,
```

Comment:
Why not use this?:

```py
vol.Inclusive(CONF_MOVE_LONG_ADDRESS, "position"): ga_list_validator,
vol.Inclusive(CONF_POSITION_ADDRESS, "position"): ga_list_validator,
```

Reply:
Inclusive would validate None or both (!XOR). 
We want one or both (OR).